### PR TITLE
Profile swipe backup

### PR DESCRIPTION
**Pull request by fegloff**

### CODE REVIEW

File: client/src/routes/home/PageNotClaimed.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
```
Comment (ahiipsa) [inline client/src/routes/home/PageNotClaimed.jsx:1]: please rename PageNotClaimed to DomainNotClaimed

Comment (ahiipsa) [inline client/src/routes/home/PageNotClaimed.jsx:1]: please rename PageNotClaimed to DomainNotClaimed

**Review by ahiipsa — CHANGES_REQUESTED** [review]

File: client/src/routes/home/PageNotClaimed.jsx (hunk 1 above)
Comment (fegloff) [reply client/src/routes/home/PageNotClaimed.jsx:1]: The component was renamed to DomainNotClaimed

**Comment by fegloff** [comment]
Request changes and files conflicts were resolved

**Pull request closed without merging**